Sentinel/Classes/Core/Internal/SentinelTabBarController.swift

```diff
@@ -45,6 +45,6 @@ private extension SentinelTabBarController {
         defer { didPreselectAction = true }
         guard !didPreselectAction else { return }
         // Preselect Tools tab
-        selectedIndex = 2
+        selectedIndex = Tabs.tools(items: []).index
```

Comment:
Even though this is much better than the previous magic number, this can be wrong if the `.tools` tab is not in the array. E.g., the index is 2, but if the `.tools` is changed to, e.g. first tab, this will not calculate the correct value.

I would suggest creating method like this;

```swift
func preselectedTabIndex(for tab: Tab, tabItems: [TabItem]) -> Int {
    return tabItems
        .enumerated()
        .first(where: {
            guard case .tools = $0.element.tab else { return false }
            return true
        })
        .map(\.offset) ?? 0
}
```

This way, we can always get the correct value for `.tools` tab.